runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/HttpConfiguration.java

```diff
@@ -39,22 +43,29 @@ public class HttpConfiguration extends Configuration
         final ConfigurationDef config = new ConfigurationDef("zilla.binding.http");
         HTTP_MAXIMUM_CONNECTIONS = config.property("maximum.connections", 10);
         HTTP_SERVER_CONCURRENT_STREAMS = config.property("server.concurrent.streams", Integer.MAX_VALUE);
+        HTTP_STREAM_INITIAL_WINDOW = config.property("stream.initial.window", 0);
         HTTP_SERVER_MAX_HEADER_LIST_SIZE = config.property("server.max.header.list.size", 8_192L);
+        HTTP_CLIENT_MAX_PUSH_PROMISE_LIST_SIZE = config.property("client.max.push.promise.list.size", 100);
+        HTTP_CLIENT_MAX_FRAME_SIZE = config.property("client.max.frame.size", 16_384);
```

Comment:
Suggest renaming to `HTP_LOCAL_` and `HTTP_REMOTE_` instead of `HTTP_CLIENT_` and `HTTP_SERVER_`?